examples/rsp-next-ts-17/next.config.js

```diff
@@ -53,7 +53,8 @@ const withTM = require("next-transpile-modules")([
   '@react-spectrum/well',
   '@spectrum-icons/illustrations',
   '@spectrum-icons/ui',
-  '@spectrum-icons/workflow'
+  '@spectrum-icons/workflow',
+  '@spectrum-icons/express'
```

Comment:
doesn't actually break with or without this change when building the app locally, not sure if this will fix, running build